css/styles.css

```diff
@@ -0,0 +1,52 @@
+body 	{
+	font-family: cursive;
+	font-size: 18px;
+	margin-top: 0px;
+	margin-right: 0px;
+	background-color: #FCDEBC;
+}
+
+nav	{
+	font-family: serif;
+	font-size:  16px;
+	text-align: right;
+	padding: 5px 15px;
+}
+
+img 	{
+	width: 50%;
+	float: right;
+	margin-left: 20px;
+}
+
+h2 {
+	margin-bottom: 0px;
+	font-style: italic;
+	font-size: 200%;
+	text-align: center;
+}
+
+h1	{
+	margin-top: -10px;
+	margin-bottom: 10px;
+	font-size: 375%;
```

Comment:
@gerriecho While something like this works, you could try messing around with `em` units instead. Perhaps something like this:
```css
font-size: 3.5em;
```
Try it and see what happens :)